### ISSUE TITLE
Areflexia of brachioradialis

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5915]
**Preferred term label:** Areflexia of brachioradialis

**Synonyms**

**Definition (free text, please give PubMed ID)** Inability to elicit brachioradialis tendon reflex.

**Parent term (use hpo.jax.org/app)** Areflexia of upper limbs HP:0012046

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
